lib/browser/chrome-extension.js

```diff
@@ -57,18 +57,23 @@ const getManifestFromPath = function (srcDirectory) {
   if (!manifestNameMap[manifest.name]) {
     const extensionId = generateExtensionIdFromName(manifest.name)
     manifestMap[extensionId] = manifestNameMap[manifest.name] = manifest
+
+    const extensionURL = url.format({
+      protocol: 'chrome-extension',
+      slashes: true,
+      hostname: extensionId,
+      pathname: manifest.devtools_page
+    })
+
     Object.assign(manifest, {
       srcDirectory: srcDirectory,
       extensionId: extensionId,
-      // We can not use 'file://' directly because all resources in the extension
-      // will be treated as relative to the root in Chrome.
-      startPage: url.format({
-        protocol: 'chrome-extension',
-        slashes: true,
-        hostname: extensionId,
-        pathname: manifest.devtools_page
-      })
+      // Chromium requires that startPage matches '([^:]+:\/\/[^/]*)\/'
+      // We also can't use the file:// protocol here since that would make Chromium
+      // treat all extension resources as being relative to root which we don't want.
+      startPage: `${extensionURL}/`
```

Comment:
The startPage can also be something like `chrome-extension://foo/foo.html`, appending `/` to it would blow devtools code.

Can probably check if `manifest.devtools_page` is empty before appending `/`.